src/pages/layouts/main.hbs

```diff
@@ -34,11 +34,17 @@
       </div>
     </div>
     <div class="wrapper">
-      {{> tag_list tags=tags}}
       <div class="content" role="main">
-        <h2>{{title}}</h2>
+        {{^if hideTitle}}
+          <h1>{{title}}</h1>
+        {{/if}}
         {{{body}}}
       </div>
+      {{#if admin}}
+        {{> admin_subnav}}
```

Comment:
I merged the main and admin templates into one—they were mostly the same save for the admin subnav showing up conditionally. I mad that condition a parameter of the main theme.